Content.Server/Dragon/DragonRiftSystem.cs

```diff
@@ -72,9 +72,8 @@ public override void Update(float frameTime)
                 comp.State = DragonRiftState.AlmostFinished;
                 Dirty(comp);
 
-                var location = xform.LocalPosition;
                 _announcer.SendAnnouncement(_announcer.GetAnnouncementId("CarpRift"), Filter.Broadcast(),
-                    "carp-rift-warning", colorOverride: Color.Red, localeArgs: ("location", location));
+                    "carp-rift-warning", colorOverride: Color.Red, localeArgs: ("location", FormattedMessage.RemoveMarkup(_navMap.GetNearestBeaconString((uid, xform)))));
```

Comment:
Iirc `RemoveMarkup` is obsolete and should be changed to `RemoveMarkupPermissive`

Reply:
Doesn't exist, I think. Ctrl+shift+f

Reply:
Nevermind, was an RT thing. Done